osu.Game/Screens/Play/Player.cs

```diff
@@ -156,7 +158,7 @@ protected virtual void PrepareReplay()
         }
 
         [BackgroundDependencyLoader]
-        private void load(AudioManager audio, OsuConfigManager config)
+        private void load(AudioManager audio, OsuConfigManager config, OsuGameBase game)
```

Comment:
```suggestion
        private void load(AudioManager audio, OsuConfigManager config, OsuGame game)
```

`[BackgroundDependencyLoader(true)]` to allow nulls